### DISCUSSION TITLE
Thanks @pwespi

### DISCUSSION THREAD

**Discussion by hecko** [capacitor-community/bluetooth-le#580] [General]
@pwespi thanks for maintaining this package! Its absolutely great - I was looking for a better way to thank you like sending you a coffee or a private email, or a donation, but could not find any contact information. 

Once again thanks for providing Capacitor community this great extension!

**Comment by mutasimissa** [comment]
Totally agree!

**Comment by pwespi** [comment]
Thank you both for your words! This means a lot! 

In case anybody still wants to give a small donation, I have this ko-fi page: https://ko-fi.com/wespiapps

Thanks again!

**Reply by hecko** [reply]
https://Ko-fi.com/home/coffeeshop?txid=8d96ac31-4aad-4633-9a7b-08009d57d021&mode=public&img=ogiboughtsomeone

Thanks!
